doc/sphinx/compiling.rst

```diff
@@ -32,69 +33,267 @@ Installation Prerequisites
+.. _sec-fedora-reqs:
 
-* For Fedora (version 22 or higher) users, the following packages should
-  be installed via the package manager::
+Fedora & RHEL
+^^^^^^^^^^^^^
 
-      gcc-c++ python scons sundials-devel blas-devel lapack-devel boost-devel
+* The following packages must be installed to build any of the Cantera modules using
+  your choice of package manager::
 
-  If your Fedora version is lower than 22, the `sundials-devel` package is not
-  available, and you should build Sundials from source.
+      gcc-c++ python scons boost-devel
 
-  Python module required packages are::
+* In addition to the general packages, building the Python 2 module also requires::
 
       python-setuptools python-devel Cython numpy
 
-* Checking out the source code from version control requires Git (install
-  ``git``).
+* In addition to the general packages, building the Python 3 module also requires::
 
-* The minimum compatible Cython version is 0.23. If your distribution does not
-  contain a suitable version, you may be able to install a more recent version
-  using `pip`.
+      python3 python3-setuptools python3-devel Cython python3-numpy
 
-* Building the Fortran interface also requires `gfortran` or another supported
-  Fortran compiler.
-* Users of other distributions should install the equivalent packages, which
-  may have slightly different names.
+.. _sec-opensuse-reqs:
+
+OpenSUSE & SEL
+^^^^^^^^^^^^^^
+
+* OpenSUSE 13.2 or newer; Leap 42.2 or newer recommended
+
+* The following packages must be installed to build any of the Cantera modules using
+  your choice of package manager::
+
+      gcc-c++ python scons boost-devel
+
+* In addition to the general packages, building the Python 2 module also requires::
+
+      python-Cython python-devel python-numpy python-numpy-devel python-setuptools
+
+* In addition to the general packages, building the Python 3 module also requires::
+
+      python-Cython python3 python3-devel python3-setuptools python3-numpy python3-numpy-devel
 
 Windows
 -------
 
-There are a number of requirements for the versions of software to install
-depending on which interfaces (Python, Matlab) you want to build and what
-architecture (32-bit or 64-bit) you want to use. See :ref:`sec-dependencies` for
-the full list of dependencies.
+General Notes
+^^^^^^^^^^^^^
+
+* SCons is only available for Python 2, so building the Python 3 module requires
+  two installations of Python (one of Python 2 and one of Python 3), even if you
+  do not intend to build the Python 2 module.
 
 * The build process will produce a Python module compatible with the version of
   Python used for the compilation. To generate different modules for other
   versions of Python, you will need to install those versions of Python and
   recompile.
+
+* The following instructions use the system-installed versions of Python, but
+  alternate installations such as the Anaconda distribution of Python can be
+  used as well.
```

Comment:
The installers from python.org that install into `C:\PythonXY` by default for all users. I clarified to say `versions of Python dowloaded from python.org`